src/database/common.c

```diff
@@ -622,20 +614,4 @@ long get_lastID(void)
 const char *get_sqlite3_version(void)
 {
 	return sqlite3_libversion();
-}
-
-// Should the long-term database be used?
-__attribute__ ((pure)) bool use_database()
-{
-	// Check if the user doesn't want to use the database and set an
-	// empty string as file name in FTL's config file or configured
-	// a maximum history of zero days.
-	if(FTLfiles.FTL_db == NULL ||
-	   strlen(FTLfiles.FTL_db) == 0 ||
-	   config.maxDBdays == 0)
-	{
-		return false;
-	}
-
-	return true;
-}
+}
```

Comment:
Just for my personal information, do you care if the file has ending newlines?

Reply:
No, I don't care and changed to whether there is a newline at the end or not are not really intended but seem unavoidable to me when working with different editors from different systems.

Reply:
Okay, good to know.